api_app/analyzers_manager/observable_analyzers/vt3_sandbox.py

```diff
@@ -0,0 +1,90 @@
+# This file is a part of IntelOwl https://github.com/intelowlproject/IntelOwl
+# See the file 'LICENSE' for copying permission.
+
+import time
+import logging
+
+import requests
+
+from api_app.exceptions import AnalyzerRunException
+from api_app.analyzers_manager.file_analyzers import vt3_scan
+from api_app.analyzers_manager.classes import ObservableAnalyzer
+
+from tests.mock_utils import patch, if_mock_connections, MockResponse
+
+logger = logging.getLogger(__name__)
+
+vt_base = "http://www.virustotal.com/api/v3/"
+
+
+class VirusTotalv3Sandbox(ObservableAnalyzer):
+    def set_params(self, params):
+        self.__api_key = self._secrets["api_key_name"]
+        # max no. of tries when polling for result
+        self.max_tries = params.get("max_tries", 10)
+        # interval b/w HTTP requests when polling
+        self.poll_distance = 30
+
+    def run(self):
+        return vt_get_behavior_summary(
+            self.__api_key,
+            self.observable_name,
+            self.observable_classification,
+            self._params,
+            self.job_id,
+            max_tries=self.max_tries,
+            poll_distance=self.poll_distance,
+        )
+
+    @classmethod
+    def _monkeypatch(cls):
+        patches = [
+            if_mock_connections(
+                patch(
+                    "requests.get",
+                    return_value=MockResponse({}, 200),
+                ),
+            )
+        ]
+        return super()._monkeypatch(patches=patches)
+
+
+def vt_get_behavior_summary(
```

Comment:
why is this not a method of the previous class?

Reply:
There was no particular reason (I'm not sure which case would be better). I thought I would be consistent with the formatting of other `VirusTotal` analyzers.

Reply:
yeah they are out the class there because they are called externally but this is not the case